components/script_bindings/codegen/codegen.py

```diff
@@ -5672,25 +5678,25 @@ def define(self) -> str:
 
 class ClassItem:
     """ Use with CGClass """
-    def __init__(self, name, visibility) -> None:
+    def __init__(self, name: str | None, visibility: str) -> None:
         self.name = name
         self.visibility = visibility
 
-    def declare(self, cgClass) -> str | None:
+    def declare(self, cgClass: CGClass) -> str | None:
```

Comment:
Instead of changing the return type, we need to suppress the error with  `# pyrefly: ignore # bad-return`.